cpg-concepts/src/main/kotlin/de/fraunhofer/aisec/cpg/graph/Extension.kt

```diff
@@ -0,0 +1,48 @@
+/*
+ * Copyright (c) 2024, Fraunhofer AISEC. All rights reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ *                    $$$$$$\  $$$$$$$\   $$$$$$\
+ *                   $$  __$$\ $$  __$$\ $$  __$$\
+ *                   $$ /  \__|$$ |  $$ |$$ /  \__|
+ *                   $$ |      $$$$$$$  |$$ |$$$$\
+ *                   $$ |      $$  ____/ $$ |\_$$ |
+ *                   $$ |  $$\ $$ |      $$ |  $$ |
+ *                   \$$$$$   |$$ |      \$$$$$   |
+ *                    \______/ \__|       \______/
+ *
+ */
+package de.fraunhofer.aisec.cpg.graph
+
+import de.fraunhofer.aisec.cpg.TranslationResult
+import de.fraunhofer.aisec.cpg.graph.concepts.Concept
+import de.fraunhofer.aisec.cpg.graph.concepts.Operation
+
+/**
+ * Retrieves a set of [Concept] nodes associated with this [TranslationResult].
+ *
+ * This property collects all overlay nodes of type [Concept].
+ *
+ * @return A set containing all [Concept] nodes found in the overlays of the [TranslationResult].
+ */
+val TranslationResult.conceptNodes: Set<Concept<*>>
```

Comment:
@oxisto can we easily extend this to also cover translation units? On second thought: do we want this for `Node`?

Same question applies to the `operationNodes` function, too.

Reply:
-> yes